bucket/qv2ray.json

```diff
@@ -0,0 +1,39 @@
+{
+    "version": "2.1.2",
+    "description": "Graphical frontend for v2ray",
```

Comment:
I suggest using "Qv2ray is a cross-platform v2ray graphical frontend written in Qt." from https://github.com/Qv2ray/Qv2ray/wiki.